bson/utils.go

```diff
@@ -0,0 +1,29 @@
+package bson
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+)
+
+func fromHex(hex string) (*big.Int, error) {
+	n, err := new(big.Int).SetString(hex, 16)
+	if !err {
+		msg := fmt.Sprintf("Cannot convert %s to int as hexadecimal", hex)
+		return nil, errors.New(msg)
+	}
+	return n, nil
+}
+
+func all(oks []bool) bool {
+	for _, ok := range oks {
+		if !ok {
+			return false
+		}
+	}
+	return true
+}
+
+func b(i int) *big.Int {
```

Comment:
Can we move this function to `encoding_test.go`? I see it's only used for test purposes.

Reply:
Yes, we can!